server/src/config/dbConnect.ts

```diff
@@ -1,6 +1,6 @@
 import { Client } from "pg";
 import { Pool } from 'postgres-pool';
-
+import createSchemas from '../models/schemas'
```

Comment:
Could you explain this line?
why are we importing this?
If we can then it will create a schema on every time database connects right?

Reply:
We can remove this initially I had confusion how are creating schema sorry you can remove that line

Reply:
Then remove this. You also remove it and update the same PR.